test/contract based tests/Lerc20Token.js

```diff
@@ -74,6 +74,46 @@ describe('Random LERC20 Token', () => {
     });
   });
 
+  describe('when transfering between users with transferFrom', () => {
+    beforeEach(async () => {
+      await lerc20Token.connect(adr.lerc20InitialHolder).transfer(adr.regularUser1.address, 100);
+      await lerc20Token.connect(adr.lerc20InitialHolder).transfer(adr.regularUser2.address, 100);
+
+      await lerc20Token.connect(adr.regularUser1).approve(adr.regularUser3.address, 50);
+      await lerc20Token.connect(adr.regularUser2).approve(adr.regularUser3.address, 50);
+    });
+
+    it('should revert if 5 minutes haven\'t passed and and it\'s a second transfer', async () => {
+      await expect(
+        lerc20Token.connect(adr.regularUser3).transferFrom(adr.regularUser2.address, adr.regularUser4.address, 5),
+      ).to.not.be.reverted;
+
+      await expect(
+        lerc20Token.connect(adr.regularUser3).transferFrom(adr.regularUser2.address, adr.regularUser4.address, 5),
+      ).to.be.revertedWith('LSS: Transfers limit reached');
+    });
+
+    it('should not revert if 5 minutes haven\'t passed but its the first transfer', async () => {
+      await expect(
+        lerc20Token.connect(adr.regularUser3).transferFrom(adr.regularUser2.address, adr.regularUser4.address, 5),
+      ).to.not.be.reverted;
+    });
+
+    it('should not revert if 5 minutes have passed', async () => {
```

Comment:
Maybe it would make sense to also check if second transfer does not revert too if 5 minutes have already passed.

Reply:
Addressed on 209788f